hack/addons/mindthegap-helm-registry/Dockerfile

```diff
@@ -3,10 +3,15 @@ FROM --platform=${BUILDPLATFORM} ghcr.io/mesosphere/mindthegap:${MINDTHEGAP_VERS
 # this gets called by goreleaser so the copy source has to be the path relative to the repo root.
 RUN --mount=source=./hack/addons/mindthegap-helm-registry/repos.yaml,target=/repos.yaml \
     ["/ko-app/mindthegap", "create", "bundle", "--helm-charts-file=/repos.yaml", "--output-file=/tmp/helm-charts.tar"]
+
+FROM --platform=${TARGETPLATFORM} ghcr.io/mesosphere/mindthegap:${MINDTHEGAP_VERSION} as mindthegap
```

Comment:
Can you recycle the from line on 2 using TARGTPLATFORM

Reply:
mindthegap from `BUILDPLATFORM` will be used to run `mindthegap create bundle` at runtime. So I think it should be `BUILDPLATFORM` in line 2.